index.html

```diff
@@ -5992,10 +5993,10 @@ <h3>Unsigned Claims</h3>
 
         <p>
 This specification allows [=credentials=] to be produced that are not secured by
-signatures or proofs of any kind. These types of [=credentials=] are often
+signatures or proofs of any kind. These [=credentials=] are often
```

Comment:
```suggestion
signatures or proofs of any kind. These class of [=credentials=] are often
```

Reply:
Merged too quickly! Should have been either --
```suggestion
signatures or proofs of any kind. These [=credential=] types are often
```

Reply:
-- or --
```suggestion
signatures or proofs of any kind. These classes of [=credential=] are often
```

Reply:
-- or perhaps --
```suggestion
signatures or proofs of any kind. Such [=credential=] are often
```